### ISSUE TITLE
Content blocked in my country

### ISSUE THREAD

**Issue by Flashwalker** [TeamNewPipe/NewPipe#3805]
Hi. I have one channel in my subscriptions which is blocked in Russia now.  
If i change default country in app settings, content still unavailable.  
But it's available then i do the same in the Youtube app.  
It would be nice to add feature to make it works. Per channel setting would be cool.

**Comment by brainchild0** [comment]
If the service is blocking content in your country, then your client would remain unable to retrieve the content as long as your IP address is associated with the country. You would have to use a VPN with an access point in an unblocked country, or an anonymous routing system, such as Tor (which the service may also block to protect against attacks), to present your device as one not associated with the blocked country.

It is not possible to achieve this effect through enhancements to a playback application such as NewPipe.

**Comment by ChiefMikeK** [comment]
## Question:  
Why then is it available via **YouTube** app?  
 but it's not available on same **IP address**, using NewPipe -> google.com api 🤔

**Comment by wb9688** [comment]
YouTube app is irrelevant, is it possible on www.youtube.com on your PC?

**Comment by Gymcap** [comment]
Had an idea for a way that NewPipe could possibily get around this.

When NewPipe encounters this "blocked by country" error, it could have an option to fall back onto invidous servers to recieve the video. 

I understand not wanting to use invidious for every request so I think only using it for when content is blocked could be a cool compromise for NewPipe.

**Comment by Flashwalker** [comment]
> YouTube app is irrelevant, is it possible on [www.youtube.com](http://www.youtube.com) on your PC?

It's available via TOR

**Comment by B0pol** [comment]
> It's available via TOR

Tor* and of course, because it's not your current IP Address. As a workaround now you can use a VPN, Orbot (use Tor for a selected app, e.g. NewPipe) on browse on invidious (add &local=true, otherwise it will still not proxy videos).

@Gymcap Invidious support is work in progress, maybe in the next major version. Using it for some issues fallback is a good idea.

**Comment by ChiefMikeK** [comment]
## I'd recommend [ProtonVPN](https://protonvpn.com/) :neckbeard:

**Comment by Stypox** [comment]
@ChiefMikeK I have nothing against ProtonVPN, but I had to hide your comment, it seemed pretty much like an adversitement ;-)

**Comment by SameenAhnaf** [comment]
Isn't #992 a duplicate of this?
